tests/foreman/cli/test_template.py

```diff
@@ -74,6 +78,75 @@ def test_update_template_1(self):
         self.assertEqual(len(result.stderr), 0)
         self.assertEqual(updated_name, result.stdout['name'])
 
+    def test_create_template_with_location(self):
+        """@Test: Check if Template with Location can be created
+
+        @Feature: Template - Create
+
+        @Assert: Template is created and new Location has been assigned
+
+        """
+
+        name = gen_string("alpha", 10)
+
+        try:
+            new_loc = make_location()
+            new_template = make_template(
+                {
+                    'name': name,
+                    'location-ids': new_loc["id"],
+                }
+            )
+        except CLIFactoryError as e:
+            self.fail(e)
+
+        self.assertIn(new_loc['name'], new_template['locations'])
```

Comment:
Nitpicks:

* Drop the extra whitespace between each hunk of code. There's no need for three blank lines in this function. It isn't that long.
* Use an exception variable name of `err` instead of `e`. `e` is so terse that newbies will be scratching their heads wondering what "e" stands for, and it makes me the reader work a little harder to decode everything.